### DISCUSSION TITLE
Week schedule possible?

### DISCUSSION THREAD

**Discussion by KaherdinTristan** [totaldebug/atomic-calendar-revive#1000] [Support] [answered]
**Hi**
Is it possible to setup Atomic Calendar Revive to show the current week as a schedule, as shown in the attached image below? If so, how? 
Best regards! 

![image](https://user-images.githubusercontent.com/3398594/233374348-1f91ae96-820c-410e-8a70-7aff93d8d362.png)

**Comment by marksie1988** [comment, marked as answer]
That's not possible no.
